spaceship/lib/spaceship/two_step_or_factor_client.rb

```diff
@@ -123,14 +123,25 @@ def handle_two_factor(response, depth = 0)
       code_length = security_code["length"]
 
       puts("")
-      puts("(Input `sms` to escape this prompt and select a trusted phone number to send the code as a text message)")
-      code_type = 'trusteddevice'
-      code = ask("Please enter the #{code_length} digit code:")
-      body = { "securityCode" => { "code" => code.to_s } }.to_json
-
-      if code == 'sms'
+      if ENV["SPACESHIP_2FA_SMS_DEFAULT_PHONE_NUMBER"]
+        puts("Environment variable `SPACESHIP_2FA_SMS_DEFAULT_PHONE_NUMBER` is set, automatically requesting 2FA token via SMS to that number")
+        puts("SPACESHIP_2FA_SMS_DEFAULT_PHONE_NUMBER = #{ENV['SPACESHIP_2FA_SMS_DEFAULT_PHONE_NUMBER']}")
+        phone_number = ENV["SPACESHIP_2FA_SMS_DEFAULT_PHONE_NUMBER"]
+        phone_id = phone_id_from_number(response.body["trustedPhoneNumbers"], phone_number)
         code_type = 'phone'
-        body = request_two_factor_code_from_phone(response.body["trustedPhoneNumbers"], code_length)
+        body = request_two_factor_code_from_phone(phone_id, phone_number, code_length)
+      else
+        puts("(Input `sms` to escape this prompt and select a trusted phone number to send the code as a text message)")
+        puts("(You can also set the environment variable `SPACESHIP_2FA_SMS_DEFAULT_PHONE_NUMBER` to automate this)")
```

Comment:
Could we put an example of the format here? It actually took me a few tries to get this working 😬 I didn't realize I had to add a `+1` to the start of my phone number. I tried adding a `1` to the start and that still didn't work 😝  So I think it might be good to put an example here for us people that don't regularly use the `+1` 😇

Reply:
This should probably be changed to a link to the documentation, that then will contain this information. (Basically: Phone number with country code (`+1`, `+49` or similar)) Having the instructions here will create a bit much output, don't you think?

Reply:
@janpio Yeah, I definitely don't want to make this output too much longer. Linking to some documentation would be 💯

Reply:
Added an output below that links to the content of this PR: https://github.com/fastlane/fastlane/pull/14524